src/main/java/Add_Command.java

```diff
@@ -0,0 +1,38 @@
+public class Add_Command extends UserCommand {
+
+    private static String desc;
+    private static String type;
+    private Task t;
+    Add_Command() {
+        super(2);
+    }
+
+    Add_Command(String type, String desc){
+        super(2);
+        this.desc = desc;
+        this.type = type;
+    }
+
+    @Override
+    public void reply(){
+        System.out.println("Sighhssss...I am your slave again?");
+    }
+
+    @Override
+    public boolean action() throws DukeException {
+        if(type.equals("todo")){
```

Comment:
There should be a space between keywords and braces, i.e. if (type.equals("todo")) { ...

Reply:
yehh thanks for highlighting.. my coding convention v bad